mountainduck/connect/index.md

```diff
@@ -0,0 +1,100 @@
+
+## Sync paused
+
+You can manually pause uploads and indexing by selecting *Pause Sync* in the submenu for the sync status. Syncing is also paused automatically when your network connection to the server is interrupted but resumed automatically when a connection is restored.
+
+```{warning}
+As long as the sync status is paused, uncached files marked as [Online Only](#online-only) can't be opened. An error message will be shown by the application attempting to open the file and a *Access Denied* notification will show up.
+```
+
+## Sync Errors
+
+Files that failed to sync get a sync error badge. You can try to repeat the failed transfer by selecting *Mountain Duck → Retry* in the context menu.
+
+### Resolve Errors
+
+If a sync error cannot be solved using *Mountain Duck → Retry* because the server does not allow the operation (i.e. due to a permission issue), you can resolve the error state on the file or folder by
+
+- Move the file or folder to another location on the volume
+- Delete the file or folder 
+
+To upload files to a target directory no longer existing on the server, you have to move the files to a location found on the server.
+
+## Sync Limitations
+
+Specific file types are excluded from sync operations per default. Files matching those criteria won't be synced from or to the server and marked with the *ignored* badge icon.
+
+### New Files and Folders
+
+New files created via context menu are created as 0-byte files and therefore not transfered to the server. The same goes for new folders with default name.
+* In Finder.app: *Untitled folder* and localized variants
+* In Windows Explorer: *New folder* and localized variants
+
+The files and folders will be synchronized after being renamed.
+
+### Files excluded by Name
+
+Some file types especially temporary files are excluded from sync processes per default. They are listed in the following [hidden configuration options](../../cyberduck/preferences.md#hidden-configuration-options).
+
+	fs.filenames.temporary.regexp=\.nfs\..* \.VolumeIcon.icns \.DS_Store .*~$ .*~\..* \._~\$.* \.~(?!lock).*[^#] .*\.tmp ~.*\.tmp .*~.*\.TMP .*\.swap .*\.swp Word\sWork\sFile.*\.tmp \._Word\sWork\sFile.*\.tmp (.*)\.sb-[\w]{8}-[\w]{6} \._.*\.(doc|docx)\.sb-.* \.TemporaryItems \.dat\.nosync.* TestFile[\d]+-[\d]+$ .*~[\w]{8}-[\w]{4}-[\w]{4}-[\w]{4}-[\w]{12}$ Adobe\ Photoshop\ [\d]+$ DBTmp.* .*\.lck [0-9A-Z]+\.tmp .*\.idlk Desktop\.ini desktop\.ini Thumbs\.db \(A\ Document\ Being\ Saved\ By\ .*\) ai[\d]{10}.* AI\ Temp.* .*\.crdownload .*\.part .*\.onetmp \.goutputstream-.* Adobe\ Premiere\ Pro\ Auto-Save
+
+### Files excluded by Pattern
+
+Files matching the following default pattern won't be synced from the server or remote storage as they are excluded from the listing process: `.`, `..`, `*/*`, `*\*`, `*:*`
+
+You can exclude additional file types from the listing by using following [hidden configuration options](../../cyberduck/preferences.md#hidden-configuration-options).
```

Comment:
You've listed only one hidden property.
"hidden configuration options" → "hidden configuration option"